spec.bs

```diff
@@ -1699,6 +1704,45 @@ Several APIs specific to fenced frames are defined on the {{Fence}} interface.
   </wpt>
 </div>
 
+<div algorithm>
+  The <dfn method for=Fence>notifyEvent(|event|)</dfn> method steps are:
+
+  1. [=Assert=]: [=this=]'s {{Document}} is [=fully active=].
```

Comment:
I think this should use `[=Document/fully active]`. See https://dontcallmedom.github.io/webdex/f.html#fully%20active%40%40Document%40dfn.

Reply:
Done.